locked-asset/proxy_dex/src/energy_update.rs

```diff
@@ -46,9 +46,6 @@ pub trait EnergyUpdateModule:
                 .get_token_attributes(token_id, token_nonce);
             energy.update_after_unlock_any(token_amount, attributes.unlock_epoch, current_epoch);
         } else if token_id == &old_locked_token_id {
-            if self.blockchain().is_smart_contract(user) {
```

Comment:
Why was this removed?

Reply:
No longer needed. Now contracts can have energy.